# Introduction

Transposable elements are classical models for epigenetic inheritance:
silent transposons can be activated and then inherited in the active
state (McClintock 1965). This inheritance can be transient, in the case
of "presetting," or it can be more permanent, with cycles of activation
and silencing lasting for several generations (McClintock 1965). The
molecular mechanisms underlying the inheritance of epigenetically
activated transposons remain obscure, although DNA methylation has been
implicated in maize (Chandler and Walbot 1986; Banks et al. 1988;
Martienssen and Baron 1994). DNA methylation can be inherited
epigenetically following DNA replication, because hemimethylated DNA is
a substrate for the DNA methyltransferase *Dnmt1* (Martienssen and Colot
2001).

In addition to DNA methylation, transposons are also subject to histone
deacetylation, histone H3 lysine-9 methylation (H3mK9), and RNA
interference (RNAi) (Rea et al. 2000; Gendrel et al. 2002; Johnson et
al. 2002; Schotta et al. 2002). These chromatin modifications are
interrelated (Martienssen and Colot 2001; Selker 2002; Sleutels and
Barlow 2002). For example, in *Neurospora* and *Arabidopsis*, DNA
methylation can be triggered by H3mK9 (Tamaru and Selker 2001; Jackson
et al. 2002; Malagnac et al. 2002) and vice versa (Johnson et al. 2002;
Soppe et al. 2002; Tariq et al. 2003). In mammals, methyl CpG-binding
proteins recruit histone deacetylase (HDAC) and histone H3 lysine-9
methyltransferase (HMT) activity (Nan et al. 1998; Fuks et al. 2003).
Additionally, the mammalian maintenance DNA methyltransferase, Dnmt1,
interacts directly with HDACs (Fuks et al. 2000). Finally, in the
fission yeast Schizosaccharomyces pombe, the RNAi machinery somehow
guides the association of H3mK9 with centromeric repeats (Volpe et al.
2002, 2003).

In mammals and S. pombe, however, there are some limitations to the
study of epigenetic regulation. For example, DNA methylation has not
been reported in fission yeast, but in the mouse, it is essential (Li et
al. 1992; Okano et al. 1999). In contrast, DNA methylation mutants are
viable and fertile in *Neurospora* and *Arabidopsis*, which permits
genetic analysis (Martienssen and Colot 2001), and a variety of genes
involved in epigenetic regulation have been identified in both
organisms.

To explore further the interrelationships between epigenetic pathways,
we have used several *Arabidopsis* mutants that affect DNA methylation,
H3mK9, and RNAi and that in some cases have been implicated in the
epigenetic regulation of transposons. For example, the chromatin
remodeling ATPase *DDM1* (open reading frame \[ORF\] At5g66750)
(Jeddeloh et al. 1999; Brzeski and Jerzmanowski 2003; Vongs et al.
1993), the *Dnmt1* homolog *MET1* (At5g49160) (Kankel et al. 2003), and
the HDAC *HDA6* (At5g63110) (Murfett et al. 2001; Aufsatz et al. 2002)
all affect silencing and DNA methylation. Further, silencing of the
*TA3*/*ATCOPIA44* retrotransposon (At1g37110) requires the DNA
methyltransferase *CHROMOMETHYLASE3* (*CMT3*) (At1g69770), and the HMT
*KRYPTONITE* (*KYP*)/*SUVH4* (At5g13960) (Bartee et al. 2001; Lindroth
et al. 2001; Jackson et al. 2002; Malagnac et al. 2002). In our studies
we have also used *sil1*, which is now known to be an allele of *hda6*
(H. Vaucheret, O. Mittelsten-Scheid, and I. Furner, personal
communication).

The mutants *cmt3* and *kyp/suvh4* were isolated as mutants that
relieved silencing imposed by long inverted repeats of the *PAI* and
*SUP* genes. A third mutant in this pathway, *argonaute4* (*ago4*)
(Zilberman et al. 2003) is related to *ARGONAUTE1* (*AGO1*) (At1g48410),
which is required for RNAi in plants, fungi, and animals (Fagard et al.
2000; Morel et al. 2002; Williams and Rubin 2002). *TA3* was unaffected
in *ago4--1*, but five of nine non-CG cytosines lost methylation in the
*MULE* DNA transposon *AtMu1*, although transcripts did not accumulate
(Zilberman et al. 2003). In *ddm1*, loss of DNA methylation is
accompanied by loss of H3mK9 and gain of H3mK4, which is correlated with
transcriptional reactivation of transposons (Gendrel et al. 2002).
Further, unmethylated centromeric repeats are inherited from *ddm1*
homozygotes (Vongs et al. 1993; Kakutani et al. 1999). This led to the
suggestion that histone modification was responsible for DNA
methylation, which could not be restored when histone modification was
lost (Gendrel et al. 2002). However, unmethylated centromeric repeats
are also inherited from *met1* homozygotes (Kankel et al. 2003), and
*met1* gametophytes (Saze et al. 2003) and the *copia*-like elements
*TA3* and *TA2* lose H3mK9 in *cmt3 met1* double mutants (Johnson et al.
2002). This led to the suggestion that DNA methylation might be
responsible for H3mK9, rather than the other way around (Gendrel et al.
2002; Richards 2002; Soppe et al. 2002; Tariq et al. 2003). Although it
is clear that epigenetic mechanisms interact, the nature of those
interactions is currently uncertain.

To explore these relationships further, we have investigated the
molecular basis for epigenetic inheritance in a representative group of
transposons by backcrossing mutants in DNA methylation, chromatin
remodeling, and histone modification to wild-type plants and
characterizing transposon chromatin modifications. Our results indicate
that the mutants fall into two groups, which might reflect the existence
of separate complexes or pathways responsible for the silencing of
different classes of transposons. Neither loss of DNA methylation nor
loss of H3mK9 can fully account for the inheritance of active
transposons. Rather, the loss of small interfering RNA (siRNA) may also
play an important role.

# Results

## Transposons Are Differentially Silenced by Chromatin Modification

We selected five class I retrotransposons and one class II DNA
transposon to assess silencing in the *Arabidopsis* ecotype Landsberg
*erecta* (L*er*) (*WT*) ([Figure 1](#pbio.0000067-g001)): the non-long
terminal repeat (LTR) retrotransposon *ATLINE1-4* (At2g01840); the
gypsy-class LTR retroelements *ATLANTYS2-1* (located between At4g03760
and At4g03770), *ATLANTYS2-2* (located between At3g43680 and At3g43690),
and *ATGP1* (At4g03650); the *copia*-like element
*ATCOPIA4/COPIA-LIKE23* (At4g16870); and the *MULE* DNA transposon
*AtMu1* (At4g08680) (Singer et al. 2001). *ATLANTYS2-1* and
*ATLANTYS2-2* were assayed with the same primer pair. In order to assess
both activation and inheritance, mutants were backcrossed to *WT*, and
F1 seed was planted and used in each assay alongside samples from selfed
mutant and *WT* parents ([Figure 1](#pbio.0000067-g001)). By assessing
transcript accumulation and association with methylated histone H3 as
well as methylated DNA in backcrossed plants heterozygous for each
mutation, we could determine whether each transposon remained silent
("cryptic"), was reversibly activated, or was heritably activated
("preset") in each mutant.

Inheritance of Transposon Activity\
Pollen from homozygous mutant plants (m/m) was crossed onto *WT* (+/+)
to generate backcrossed BC heterozygous seed (m/+). The parents were
also self-pollinated as a control. Each class of progeny was then tested
for expression of transposon mRNA, loss of DNA methylation, and changes
in histone H3 methylation. Accumulation of transposon mRNA (+) or lack
thereof (−) in each progeny genotype was used to determine whether the
elements were silent ("cryptic"), reversibly activated, or heritably
activated ("preset").

In *WT*, transcripts were low or undetectable by PCR amplifying
reverse-transcribed cDNA (RT--PCR), and these loci were associated with
elevated levels of H3mK9 and reduced levels of H3mK4 according to
chromatin immunoprecipitation (ChIP) analysis. The transposons were also
heavily methylated when assayed by modified cytosine restriction McrBC
digestion, which cuts DNA at methylated cytosine residues, preventing
PCR amplification ([Figure 2](#pbio.0000067-g002)C), or by DNA gel blot
analysis using HpaII and MspI, which are sensitive to both CG and CNG
methylation and to CNG methylation alone, respectively ([Figure
3](#pbio.0000067-g003)). Transcripts, unmethylated DNA, and H3mK4 could
be detected in the mutants (see [Figure 2](#pbio.0000067-g002)) and were
indicative of the inheritance of activated transposons in backcrossed
plants in all cases except *ATGP1*, which had substantial levels of
H3mK4 in *WT* plants. Methylated DNA and H3mK9 were also measured, but
could not be used to assess inheritance, as these were also inherited
from silent elements in the *WT* parent.

Inheritance of Transposon Modification\
Reverse-transcribed cDNA (A), ChIP (B), and McrBC-digested genomic DNA
(C) were amplified by PCR using primers from five retroelements and one
DNA transposon in mutant (m/m) and backcrossed plants (m/+). Primers
corresponded to transcribed ORFs for each element except for *AtMu1*
ChIP, which was done on the terminal inverted repeat (TIR). For
*ATLANTYS2*, the larger product is *ATLANTYS2-1* and smaller product is
*ATLANTYS2-2*. Input RNA was normalized for each genotype using actin
primers.\
(A) Mock RT--PCR was performed without reverse transcriptase (−RT) using
primers specific for the Cen180 repeat, which can detect trace amounts
of contaminating DNA due to its high-copy number.\
(B) ChIP was performed with antibodies recognizing dimethyl lysine-9
(K9) and dimethyl lysine-4 (K4) of histone H3 along with no antibody
(na) and total (T) DNA controls. ChIP analysis for *AtMu1* and
*ATCOPIA4* was performed using reduced cycles of PCR and Southern
blotting (see [Materials and Methods](#s4)).\
(C) McrPCR was carried out on untreated (−) and McrBC-treated (+) DNA
(see [Materials and Methods](#s4)).

Southern Blot Analysis\
(A and B) Genomic DNAs prepared from 4-wk-old plants of the indicated
mutant and backcrossed (m/+) genotypes were digested with either HindIII
and HpaII (left) or HindIII and MspI (right) and used for Southern blot
analysis with a probe specific to the DNA transposons *AtMu1* and the
retrotransposon *ATCOPIA4*. The L*er* genotype is shown. DNA methylation
loss for each element within the mutants and their backcrosses is
indicated by loss of band intensity relative to *WT* as indicated by the
arrows or brackets.\
(C) Genomic DNAs from the same genotypes in (A) and (B) were digested
with either HpaII (left) or MspI (right) and used for Southern blot
analysis with a probe specific to the *ATLINE1-4* element. The probe
corresponds to a region flanked on both sides by more than five
HpaII/MspI sites within 6 kb. Thus, fragment sizes generated upon
digestion of the genotypes tested varied owing to a number of potential
methylation changes. The fragments within the brackets depict
significant changes in methylation between the genotypes.

Transcripts from all six transposons accumulated in *ddm1*, accompanied
by loss of DNA methylation and H3mK9 and gain of H3mK4 (see [Figure
2](#pbio.0000067-g002)). Following backcrosses, each of the six
transposons remained hypomethylated in *ddm1/+* plants. They were
associated with H3mK4, and transcripts could still be detected. All six
elements were also activated in *met1*, but *ATLINE1-4* and *ATGP1* were
partially or completely resilenced in *met1/+* backcrosses,
respectively, and *ATLINE1-4* did not retain H3mK4 (see [Figure
2](#pbio.0000067-g002)). Interestingly, *ATLINE1-4* retained H3mK9 in
*met1* (see [Figure 2](#pbio.0000067-g002)B), although it was
hypomethylated ([Figure 3](#pbio.0000067-g003)). In *sil1*, transcripts
from five of the six elements also accumulated. Three of these elements,
*AtMu1*, *ATCOPIA4*, and *ATLINE1-4*, lost DNA methylation along with
H3mK9 (see [Figure 2](#pbio.0000067-g002); [Figure
3](#pbio.0000067-g003)), and two of them, *ATCOPIA4* and *ATLINE1-4*,
gained H3mK4 (*AtMu1* already had substantial levels in *WT*). Histone
H3 and DNA methylation were unchanged in the high-copy *ATLANTYS2* and
*ATGP1* elements, perhaps because only a subset of elements was
transcriptionally activated. In backcrossed *sil1/+* plants,
transcripts, DNA hypomethylation, and H3mK4 were restored to *WT*
levels, unlike in *ddm1/+* and *met1/+*, indicating these changes were
reversible and not "preset." Thus, *SIL1* can silence transposons de
novo when introduced in backcrossed plants (see [Figure
1](#pbio.0000067-g001)), unlike *DDM1* and *MET1*. This was unexpected,
as the molecular changes observed in *ddm1*, *met1*, and *sil1* were
comparable.

*kyp* and *cmt3* had much weaker effects on transposon activation,
despite widespread loss of H3mK9 in *kyp*, and on CNG methylation in
*cmt3* (see [Figure 2](#pbio.0000067-g002); [Figure
3](#pbio.0000067-g003)). Specifically, in *cmt3*, *ATLINE1-4* was
heritably activated and *ATCOPIA4* accumulated low levels of transcript.
In *kyp*, only *ATCOPIA4* was activated and associated with high levels
of H3mK4. CNG methylation was lost and not restored in *kyp/+*, as in
*cmt3/+* ([Figure 3](#pbio.0000067-g003)), although sensitivity to McrBC
was unaffected, presumably due to methylation of non-CNG sequences (see
[Figure 2](#pbio.0000067-g002)). The gypsy-class elements *ATLANTYS2*
and *ATGP1* remained silent in both mutants. Thus, while loss of CG and
CNG methylation, loss of H3mK9, and gain of H3mK4 accompany transposon
activation, none of these can reliably predict their subsequent
inheritance.

## The Role of RNAi

In Caenorhabditis elegans and *Drosophila*, which lack DNA methylation,
transposon silencing is maintained in the germline by RNAi (Plasterk and
Ketting 2000; Aravin et al. 2001), and we examined whether RNAi impacts
transposon silencing in *Arabidopsis* using a strong allele of *ago1*,
*ago1-9* (C. Kidner and R. Martienssen, unpublished data ). Strong and
weak alleles of *ago1* are defective in transgene silencing and
methylation (Fagard et al. 2000; Morel et al. 2002), they have strong
developmental phenotypes, and they are sterile in L*er* (Fagard et al.
2000). In *ago1-9,* only *ATCOPIA4* was activated, accompanied by loss
of H3mK9 and gain of H3mK4, but DNA methylation was unaffected (see
[Figure 2](#pbio.0000067-g002); data not shown). *ATCOPIA4* is located
in a disease-resistance gene cluster on the long arm of Chromosome 4
that undergoes frequent epimutation in *ddm1* inbred strains (Stokes and
Richards 2002). The DNA transposon *AtMu1* was weakly transcribed in
*WT* plants (Singer et al. 2001), making its activation in *ago1*,
*cmt3*, and *kyp* difficult to detect. However, DNA methylation was lost
from *AtMu1* in each of these three mutants.

Thus, *ago1* resembles *kyp*, in having relatively minor effects on
transposon silencing. One explanation is genetic redundancy. There are
ten *AGO* genes in the *Arabidopsis* genome, and a mutant allele of
*ago4* also has a modest impact on *AtMu1* methylation (Zilberman et al.
2003). Redundancy cannot be the entire explanation, however, because we
found other similarities between *ago1*, *cmt3*, and *kyp*. Using
primers from 24 retrotransposons and 18 DNA transposons from the
heterochromatic knob (Gendrel et al. 2002), we found that almost all of
them remained silent in *ago1* and *cmt3* (data not shown). However,
*ATENSPM5* (At4g03910) was weakly activated in *ago1* and behaved
exactly like *ATCOPIA4* in the other mutants (data not shown). In
contrast, more than half of the transposons in the knob were strongly
activated in both *ddm1* and *met1* (Gendrel et al. 2002; Tariq et al.
2003). This indicated that silencing mediated by *AGO1*, *KYP*, and
*CMT3* is distinct from silencing mediated by *DDM1* and *MET1*.

We looked for siRNA in each of the mutants ([Figure
4](#pbio.0000067-g004)). Long siRNA (25 nt) is a hallmark of transposons
targeted by RNAi (Llave et al. 2002) and is presumably the product of a
*DICER*-like (*DCL*) enzyme specialized for this purpose (Hamilton et
al. 2002). As a control, a 21 nt microRNA (miRNA) derived from hairpin
precursors (Rhoades et al. 2002) accumulated to normal levels in all
genotypes examined. miRNA is the product of *DICER-LIKE1* (*DCL1*)
(At1g01040), and *dcl1-9* mutants (Jacobsen et al. 1999) had no effect
on any of the transposons tested (data not shown). While we could not
detect siRNA corresponding to *ATCOPIA4*, *ATLINE1-4*, or *ATLANTYS2*,
25 nt siRNAs corresponding to *AtMu1* and *ATGP1*, as well as the short
interspersed nuclear retroelement *AtSN1* (Hamilton et al. 2002),
accumulated in *WT* plants. These siRNAs accumulated to normal levels in
*sil1*, *kyp*, and *cmt3*, but *AtMu1* and *AtSN1* were absent or nearly
so in *met1* and *ddm1* ([Figure 4](#pbio.0000067-g004); data not
shown). In contrast, siRNA from the LTR and coding sequence of *ATGP1*
was normal in *met1* and *ddm1* ([Figure 4](#pbio.0000067-g004); data
not shown). siRNA in *ago1* had the opposite pattern: transposon siRNA
accumulated to normal levels except for *ATGP1*, which had reduced
levels ([Figure 4](#pbio.0000067-g004)). This indicates a role for
*MET1* and *DDM1* in siRNA accumulation and a role for siRNA in
epigenetic inheritance.

siRNA Differentially Accumulates in Chromatin Mutants\
siRNA Northern blots were hybridized with sense RNA probes for each of
the transposons indicated in (A) and (B) in order to detect 25 nt
antisense siRNA from each of the sequences tested. *AtMu1* is single
copy so that autoradiographic exposure was increased substantially. A 22
oligonucleotide size marker is indicated, and the 21 nt miRNA *miR-171*
was used as a loading control. It is unaffected in the mutants tested.

# Discussion

## Two Distinct Mechanisms Silence Transposons

Each of the mutants described here has been previously shown to impact
transposon methylation, transcription, and H3mK9 accumulation (Gendrel
et al. 2002; Johnson et al. 2002; Tariq et al. 2003). However, different
transposons were used in each case, and inheritance of activated
transposons was not tested. For example, *TA3* (*ATCOPIA44*) is
activated in *cmt3* (Johnson et al. 2002). *TA3* is closely related to
*ATCOPIA4* (*copia* superfamily 6), which we show is also affected, but
gypsy-class retrotransposons are not affected at all, and class II DNA
transposons are only weakly affected. In another example, *ATLANTYS2*
and *ATENSPM2* (as well as the defective *ATCOPIA* and *VANDAL* elements
Ta2 and At4g03870) were shown to lose H3mK9 in a null allele of *met1*,
leading to the conclusion that CG methylation is required for HMT
activity (Tariq et al. 2003). Here we demonstrate that, while
*ATLANTYS2-2*, *AtMu1*, and *ATCOPIA4* do indeed lose H3mK9 in *met1-1*,
*ATLANTYS2-1*, *ATLINE1-4*, and *ATGP1* do not lose H3mK9, despite loss
of CG methylation.

We have taken a genetic approach to dissecting transposon regulation. By
examining representative transposons of each class in each mutant, we
demonstrate first that transposons differ in their regulation. Next, we
show that the mutants can be grouped according to their pattern of
transposon regulation, revealing two distinct mechanisms of transposon
silencing ([Figure 5](#pbio.0000067-g005)A). *kyp* resembles *ago1-9* in
that it is only required to silence a subset of transposons, even though
*kyp* results in widespread loss of H3mK9. *met1* and *ddm1* resemble
each other and *sil1* more closely than *cmt3*, *ago1*, and *kyp*. In
*sil1*, H3mK9 is lost, but unlike in *kyp*, most of the elements are
derepressed. *SIL1* encodes the HDAC HDA6 (H. Vaucheret, O.
Mittelsten-Scheid, and I. Furner, personal communication), which has
been implicated in posttranscriptional gene silencing (Murfett et al.
2001) as well as in RNA-directed DNA methylation (Aufsatz et al. 2002).

Model of Transposon Silencing Complexes\
(A) DDM1, MET1, and SIL1 are all required for transposon silencing and
may interact. MET1 and DDM1 are also required for siRNA accumulation
(shown in red). AGO and KYP have similar effects on transposon
activation and may also interact. They impact DNA methylation via CMT3
(Cao and Jacobsen 2002; Jackson et al. 2002).\
(B) siRNA, histone H3 methylation, and DNA methylation interact to
silence transposons. Silencing is maintained by the MET1/DDM1/SIL1
complex. A possible network is shown.

There are two formal explanations when mutants in different genes have
similar phenotypes. The first is that the gene products interact in a
complex, so that removal of any one will disrupt the function of the
others. The second explanation is that the genes interact in a pathway,
so that one is upstream of the other. We propose a model taking both of
these possibilities into account ([Figure 5](#pbio.0000067-g005)B).
MET1, DDM1, and SIL1 may act together in a complex, accounting for loss
of histone modification in *met1* mutants and loss of DNA methylation in
*ddm1* and *sil1*. This is also consistent with gain of H4K16
acetylation in *ddm1* chromocenters (Soppe et al. 2002). In contrast,
*KYP* and *AGO1* affect only a subset of transposons and may interact in
a separate complex ([Figure 5](#pbio.0000067-g005)A). Their effects on
DNA methylation are mediated by CMT3, which utilizes H3mK9 as a guide
(Cao and Jacobsen 2002; Jackson et al. 2002). There are precedents for
each complex. The human nucleolar chromatin remodeling complex, NoRC,
includes a SWI/SNF chromatin remodeling ATPase (Snf2h) as well as the
RNA-binding protein TIP-5, the DNA methyltransferase Dnmt1, and HDAC1
(Santoro et al. 2002). DDM1 is strongly required for rDNA methylation,
supporting this idea (Vongs et al. 1993). In S. pombe, *ago1*^+^ and the
HMT *clr4*^+^ each effect H3mK9 as well as RNAi, indicating their
products may also interact (Volpe et al. 2002; Schramke and Allshire
2003).

## Transposon Silencing Complexes Interact via siRNA and Histone Modification

Although the mutants fall into separate groups, the *ATCOPIA4* and
*ATENSPM5* transposons silenced by *KYP*, *AGO1*, and *CMT3* are also
silenced by *DDM1*. Therefore, the two complexes act in a common
pathway. One common intermediate is siRNA. There are ten *AGO*-like
genes in *Arabidopsis*, so different transposons may utilize different
KYP/AGO complexes. These complexes presumably interact with siRNA (Caudy
et al. 2002). siRNA is stabilized by DDM1/MET1. If siRNAs were shared
between the two complexes, this would account for the complementary
accumulation of siRNA in *ago1* and *met1*, in that siRNAs that
accumulate in *met1* fail to accumulate in *ago1* and vice versa (see
[Figure 4](#pbio.0000067-g004)).

Another common intermediate is histone H3 modification. H3mK9 by KYP may
depend on deacetylation by SIL1, accounting for the observation that
H3mK9 depends on both complexes. These changes in histone modification
impact CXG methylation indirectly via CMT3 (Cao and Jacobsen 2002;
Jackson et al. 2002). However, while both *sil1* and *kyp* impact H3mK9,
only *sil1* has a major effect on transposon activation. The
MET1/DDM1/SIL1 complex can maintain silencing in the absence of KYP, but
KYP cannot maintain silencing in the absence of DDM1, MET1, or SIL1. The
most likely explanation is that DDM1 and MET1 influence histone
modification through SIL1 ([Figure 5](#pbio.0000067-g005)) rather than
directly via KYP, as previously proposed (Johnson et al. 2002). These
results implicate the gain of H3mK4, rather than the loss of H3mK9, as
being important for transposon activation. It is possible, therefore,
that H3mK4 is specifically excluded by DDM1 remodeling and that loss of
H3mK9 in *ddm1* mutants is indirect (Gendrel et al. 2002).

## Silencing of Active Transposons via siRNA

Active retrotransposons are epigenetically inherited from the
methyltransferase mutants *met1* and *cmt3*. An attractive mechanism
accounting for this inheritance is that loss of DNA methylation cannot
be restored by maintenance methyltransferase (Tariq et al. 2003).
However, the loss of DNA methylation in *sil1* is comparable to *cmt3*
and *met1*, and yet active transposons are readily silenced in *sil1/+*
backcrosses. One difference between these mutants is that *met1* does
not accumulate siRNA corresponding to *AtSN1* or *AtMu1*, resembling in
this respect the silencing mutants *ago4* and *sde4* (Hamilton et al.
2002; Zilberman et al. 2003). siRNA accumulates normally in *sil1*. Loss
of siRNA is not due to silencing of these transposons, as *AtMu1* is
activated in *sil1*, *ddm1*, and *met1*. In contrast, *ATGP1* siRNA
levels are unaffected and *ATGP1* is silenced in *met1/+*. Further, the
only elements that retained H3mK9 in *met1* (*ATLANTYS2-1*, *ATLINE1-4*,
and *ATGP1*) exhibited at least some resilencing in *met1/+*.

Thus, *MET1* may require siRNA for silencing de novo. CMT3 may also
require siRNA: *ATLINE1-4* was not silenced when *cmt3* was backcrossed
to *WT*, but *PAI2* and *SUP* genes activated in *cmt3* could be
silenced by complementation with *CMT3* transgenes (Bartee et al. 2001;
Lindroth et al. 2001). Complementation was in the presence of an
inverted repeat, which could provide siRNA *in trans*. We have not been
able to detect siRNA from *ATLINE1-4*. If siRNA guides silencing by
*MET1*, it would have to act *in cis*, as it is provided from the *WT*
parent in *met1/+* backcrossed plants. siRNA contributed *in trans*
might eventually reestablish silencing in subsequent generations,
resembling the presetting and cycling of transposon activity in maize.
Such long-term consequences of silencing deserve further investigation.

# Materials and Methods

## 

### Plant material

All plants were of the Landsberg *erecta* (L*er*) ecotype and grown in a
greenhouse under long days. *ddm1-2* and *met1-1* were introgressed into
L*er* from Columbia by backcrossing five to eight times and inbreeding
by single-seed descent for two (*met1-1*) or three (*ddm1-2*)
generations (Singer et al. 2001; Kankel et al. 2003). *cmt3-m5662* is a
*DsE* enhancer trap insertion in the 16th exon (ET5662;
<http://genetrap.cshl.org>), which blocks *CMT3* transcription (data not
shown), and was inbred for two generations. *sil1* (Furner et al. 1998),
*dcl1-9* (Jacobsen et al. 1999), *ago1-9* (C. Kidner and R. Martienssen,
unpublished data), and *kyp-2* (Jackson et al. 2002) were as previously
described. Backcrosses onto L*er*, serving as females, were performed
with mutant pollen, and progeny were pooled for analysis.

### Expression analysis

Total RNA was extracted with Trizol reagent (Life Technologies,
Carlsbad, California, United States) from 4-wk-old plants. Contaminating
DNA was removed with RNase-free DNase (RQ1-DNase; Promega, Madison,
Wisconsin, United States), and reactions were performed in 25 μl using
100 ng of RNA and the Qiagen (Valencia, California, United States)
One-Step RT--PCR kit. Input RNA was normalized for each genotype using
actin primers and dilutions of wild-type RNA (Figure2A). Mock RT--PCR
was performed without reverse transcriptase using primers specific for
the Cen180 repeat, which can detect trace amounts of contaminating DNA
due to its high-copy number. RT--PCR conditions were as follows: +RT:
50°C for 30 min, 95°C for 15 min, 35 times (94°C for 30 s, 60°C for 30
s, 72°C for 2 min), 72°C for 10 min; −RT: 4°C for 30 min, 95°C for 15
min, 35 times (94°C for 30 s, 60°C for 30 s, 72°C for 2 min), 72°C for
10 min. The amplified DNA was visualized on a 2.0% agarose gel stained
with ethidium bromide. *AtMu1* is weakly expressed in *WT* (Singer et
al. 2001), and the highest level detected is shown in [Figure
2](#pbio.0000067-g002)A. In multiple replicates, *AtMu1* was
consistently up-regulated in *sil1*.

siRNA was purified by clearing larger transcripts with PEG precipitation
and was detected using 15% polyacrylamide gel blots as described (Dalmay
et al. 2000). RNA---30 μg (*ATSINE*, *miR-171*, *ATGP1*) or 60 μg
(*AtMu1*)---was loaded per lane and RNA gels were transferred onto
Hybond N^+^ (Amersham, Little Chalfont, United Kingdom) nitrocellulose
membranes. Riboprobe templates were generated by PCR from genomic DNA
using primers with a T3 promoter sequence (AATTAACCCTCACTAAAGGGAGA).
Sense riboprobes were generated by in vitro transcribing of each DNA
template with an Ambion (Austin, Texas, United States) Maxiscript T3 in
vitro transcription kit. miRNA probes were prepared by end-labeling
antisense oligonucleotides using T4 polynucleotide kinase (New England
Biolabs, Beverly, Massachusetts, United States). An end-labeled 22 nt
RNA was used as a size marker and its position is indicated in [Figure
4](#pbio.0000067-g004). *AtMu1* siRNA analysis was repeated in two
independent experiments to verify results from this single-copy element
where only *met1* and *ddm1* exhibited loss of siRNA. All other
sequences tested were multicopy. Therefore, our detection of siRNAs
reflects the entire transposon population.

### Chromatin immunoprecipitation

ChIP was carried out as described elsewhere (Gendrel et al. 2002) using
4-wk-old soil-grown plants and histone H3 anti-dimethyl lysine-9 or
anti-dimethyl lysine-4 antibodies (Upstate Technologies, Avon, New York,
United States). Precipitated DNA was resuspended in 100 μl for PCR
analysis. An equal amount of chromatin was mock-precipitated without
antibody, while a small aliquot of sonicated chromatin was reverse
cross-linked, resuspended in 100 μl, diluted, and used as the total
input DNA control. PCRs were performed in 25 μl with 1 μl of
immunoprecipitated DNA. PCR conditions were as follows: 94°C for 3 min,
35 times (94°C for 20 s, 60°C for 30 s, 72°C for 1.5 min), 72°C for 10
min. The amplified DNA was visualized on 2% agarose gels stained with
ethidium bromide. For *AtMu1* and *ATCOPIA4*, three different cycle
numbers were compared (19, 21, and 23 cycles) by PCR and analyzed by
Southern blots. Samples from each genotype were normalized to each other
by amplifying dilutions of total input DNA with each of the primer
pairs. Qualitative data were then obtained by comparing amplification
with each set of primer pairs within the same ChIP extraction, which
served as internal controls. In this way, control primers such as actin,
whose association with lysine-9 is unclear, could be avoided. In all
cases, mock precipitation with no antibody yielded little or no product.

### Primers and PCR

Primers for RT--PCR, McrPCR, Southern blot probes, and riboprobes were
selected using Primer3
(<http://www-genome.wi.mit.edu/cgibin/primer/primer3_www.cgi>) and
BLASTN. Primer sequences are available upon request. All primer pairs
were predicted to amplify a single product in the *Arabidopsis* genome
except for *ATLANTYS2* and *ATGP1. ATGP1* is also highly repetitive and
therefore multiple elements are detected by PCR. DNA methylation was
assessed by PCR amplification of DNA that had been pretreated with
McrBC, a methylation-dependent restriction enzyme that restricts
purine-C~methyl~ half-sites separated by 80 bp up to 3 kb (New England
Biolabs). Successful amplification after digestion indicates lack of
methylation. Genomic DNA (2 μg) from each genotype was digested for 0
min, 25 min, and 8 h, followed by heat inactivation. Template (60 ng)
was then amplified using the PCR for 24 cycles, as described elsewhere
(Rabinowicz et al. 2003).

### Informatics

Transposons were annotated according to TIGR v3 (with supplementary
information from RepBase), with the corresponding open reading frame
(ORF) designations: *ATLINE1-4* (At2g01840), *ATLANTYS2-1/Cinful-1*
(At4g03760--At4g03770) and *ATLANTYS2-2/Cinful-2*
(At3g43680--At3g43690), *ATGP1* (At4g03650), *ATCOPIA4/COPIA-LIKE23*
(At4g16870), *AtMu1* (At4g08680), *TA3/ATCOPIA44* (At1g37110), and
*ATENSPM5* (At4g03910).

We thank Eric Richards, Steve Jacobsen, Caroline Dean, Catherine Kidner,
and Ian Furner for providing inbred strains; Amy Caudy, Greg Hannon, and
Mike Ronemus for advice on siRNA gels; and Ortrun Mittelsten-Scheid,
Herve Vaucheret, and Ian Furner for permission to cite unpublished work
on *SIL1*. This work was supported by a grant from the National Science
Foundation (DBI-07774).

*AGO1*

:   *ARGONAUTE1*

*AGO4*

:   *ARGONAUTE4*

ChIP

:   chromatin immunoprecipitation

*CMT3*

:   *CHROMOMETHYLASE3*

*DCL1*

:   *DICER-LIKE1*

*DDM1*

:   deficient in DNA methylation 1

H3mK9

:   histone H3 lysine-9 methylation

H4K16

:   histone H4 at lysine-16

HDAC

:   histone deacetylase

HMT

:   histone H3 lysine-9 methyltransferase

K4

:   dimethyl lysine-4

K9

:   dimethyl lysine-9

*KYP*

:   *KRYPTONITE*

L*er*

:   Landsberg *erecta*

LTR

:   long terminal repeat

miRNA

:   microRNA

NoRC

:   nucleolar chromatin remodeling complex

ORF

:   open reading frame

RNAi

:   RNA interference

RT--PCR

:   PCR amplifying reverse-transcribed cDNA

siRNA

:   small interfering RNA

Snf2h

:   SWI/SNF chromatin remodeling ATPase

TIR

:   terminal inverted repeat

WT

:   wild-type

[^1]: ZL, BM, and RM conceived and designed the experiments. ZL, BM, and
    CY performed the experiments. ZL, BM, and CY analyzed the data. ZL,
    BM, and TS contributed reagents/materials/analysis tools. ZL, BM,
    and RM wrote the paper.

[^2]: The authors have declared that no conflicts of interest exist.
